libminifi/test/CPPLINT.cfg

```diff
@@ -1,4 +1,4 @@
-filter=-build/namespaces
+filter=-build/namespaces,-whitespace/braces
```

Comment:
There were two false positives in PersistableKeyValueStoreServiceTest.cpp and UnorderedMapKeyValueStoreServiceTest.cpp, "missing whitespace before {" at Catch::clara::Opt construction. The google style guide allows uniform initialization without a space before the brace.